eppo_client/types.py

```diff
@@ -3,5 +3,5 @@
 ValueType = Union[str, int, float, bool]
 AttributeType = Union[str, int, float, bool, None]
 ConditionValueType = Union[AttributeType, List[AttributeType]]
-SubjectAttributes = Dict[str, AttributeType]
+AttributesDict = Dict[str, AttributeType]
```

Comment:
Renamed this to re-use the type

Reply:
Nice, I did the same in Java calling it "EppoAttributes"